wordpress-mobile/gutenberg-mobile
# Add Buildkite integration with publish to s3 plugin

This PR ports Android artifact publishing from CircleCI to Buildkite. It also integrates Android build file changes for `react-native-bridge` & `react-native-aztec` projects from https://github.com/WordPress/gutenberg/pull/33441.

**Note that https://github.com/WordPress/gutenberg/pull/33441 needs to be merged and the gutenberg submodule needs to be updated before this PR can be merged.**

**To test:**
1. Test [WPAndroid](https://github.com/wordpress-mobile/WordPress-Android/) by updating `ext.gutenbergMobileVersion` in `build.gradle` to `3733-9e56603c19bf429f66b231806717f825bd90bf65`
2. Test the demo app and verify that it works as expected
3. Test [WPAndroid](https://github.com/wordpress-mobile/WordPress-Android/) with composite build

~I've been unable to run Test (2) and Test (3) as I am getting JS errors. I'd appreciate some help from a gutenberg-mobile developer.~ It turns out I just didn't initialize the `block-experiments` git submodule 🤦 Everything works fine now.
